src/main/kotlin/com/slack/moshi/interop/gson/ClassCheckers.kt

```diff
@@ -52,6 +53,16 @@ public object JsonClassClassChecker : ClassChecker {
   override fun toString(): String = "JsonClassClassChecker"
 }
 
+/** Checks for classes annotated with GSON's [JsonAdapter] annotation. */
+public object JsonAdapterAnnotationChecker : ClassChecker {
```

Comment:
We have inconsistent naming between `JsonAdapterAnnotationChecker` and `JsonClassClassChecker`. Either we rename this `JsonAdapterClassChecker` or we rename them both to `JsonAdapterAnnotationClassChecker` format. I prefer the second one actually - it's clear we're checking for an annotation, but also consistent with other `ClassChecker` naming.

Reply:
363988a